crates/burn-tensor/src/tests/ops/repeat.rs

```diff
@@ -36,90 +36,95 @@ mod tests {
 
     #[test]
     fn should_support_int_repeat_ops_one_dimension() {
-        let data = TensorData::from([[0, 1, 2]]);
+        let data = TensorData::from([[0i32, 1i32, 2i32]]);
         let tensor = Tensor::<TestBackend, 2, Int>::from_data(data, &Default::default());
 
         let output = tensor.repeat(&[(0, 4)]);
-        let expected = TensorData::from([[0, 1, 2], [0, 1, 2], [0, 1, 2], [0, 1, 2]]);
+        let expected = TensorData::from([
+            [0i32, 1i32, 2i32],
+            [0i32, 1i32, 2i32],
+            [0i32, 1i32, 2i32],
+            [0i32, 1i32, 2i32],
+        ]);
 
         output.into_data().assert_eq(&expected, false);
     }
 
     #[test]
     fn should_support_float_repeat_repeating_on_many_dimensions() {
         let data = TensorData::from([
-            [[1.0, 2.0], [3.0, 4.0]],
-            [[5.0, 6.0], [7.0, 8.0]],
-            [[9.0, 10.0], [11.0, 12.0]],
-            [[13.0, 14.0], [15.0, 16.0]],
+            [[1.0f32, 2.0f32], [3.0f32, 4.0f32]],
+            [[5.0f32, 6.0f32], [7.0f32, 8.0f32]],
+            [[9.0f32, 10.0f32], [11.0f32, 12.0f32]],
+            [[13.0f32, 14.0f32], [15.0f32, 16.0f32]],
```

Comment:
No need to change this but it's sufficient to specify type once (at the beginning) and rust can infer the types.